src/Network/Network.Test/ScenarioTests/NetworkManagerTests.ps1

```diff
@@ -698,4 +698,275 @@ function Test-NetworkManagerResourceMinimumParameterCreate
         # Cleanup
         Clean-ResourceGroup $rgname
 	}
+}
+
+<#
```

Comment:
there is a test in this file named 'Test-NetworkManagerResourceMinimumParameterCreate', whose purpose is to test each Create command **with only required parameters**. Can you add routing and nsg resources to that?

Reply:
Updated